### DISCUSSION TITLE
classes/class_reflectionprobe

### DISCUSSION THREAD

**Discussion by giscus** [godotengine/godot-docs-user-notes#33] [User-contributed notes]
# classes/class_reflectionprobe

Inherits: VisualInstance3D< Node3D< Node< Object Captures its surroundings to create fast, accurate reflections from a given point. Description: Captures its surroundings as a cubemap, and stores v...

https://docs.godotengine.org/en/latest/classes/class_reflectionprobe.html

**Comment by radiantgurl** [comment]
If you want a quick update, you can set update_mode to update always to force an instant update then turn it back to be able to see the changes immediately, as Godot usually takes a second to update reflection probes with UPDATE ONCE. This is useful for cases where you want quick updates but you still dont want it to always update.
